src/raftstore/store/util.rs

```diff
@@ -323,6 +338,32 @@ impl fmt::Debug for Lease {
     }
 }
 
+/// Parse data of entry `index`.
+// TODO: make sure received entries are not corrupted
+// If this happens, TiKV will panic and can't recover without extra effort.
```

Comment:
Could you add a panic section? https://doc.rust-lang.org/book/first-edition/documentation.html#special-sections